internal/deviations/deviations.go

```diff
@@ -224,4 +229,6 @@ var (
 		"Device skip isis restart-suppress check if value is true, Default value is false")
 
 	MacAddressMissing = flag.Bool("deviation_mac_address_missing", false, "Device does not support /system/mac-address/state.")
+
+	schedulerInputWeightLimit = flag.Bool("deviation_scheduler_input_weight_limit", false, "Device does not support weight above certain limit")
```

Comment:
Can you please add more context on the deviation in the comments

Reply:
Hi Sach, 
I have added some context in the description, let me know if more detailed description is expected
Thanks,
Charan